grpcx/server.go

```diff
@@ -36,6 +36,7 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 		respMsg, _ := json.Marshal(resp)
 
 		l.Info("gRPC Message",
```

Comment:
@sp-manuel-peralta do you think that makes sense to keep this message or should we change it?

Reply:
I would change it, but not sure for what XD